src/converse-chatview.js

```diff
@@ -345,6 +346,17 @@ converse.plugins.add('converse-chatview', {
                 'paste .chat-textarea': 'onPaste',
                 'dragover .chat-textarea': 'onDragOver',
                 'drop .chat-textarea': 'onDrop',
+                'click .toggle-download': 'downloadAttachements',
+            },
+
+            downloadAttachements () {
+                var downloadable = this.el.getElementsByClassName('downloadable');
+                if (downloadable.length === 0) {
+                    this.showHelpMessages([__('There are no files to download.')], 'error');
+                    return;
+                }
+                _converse.multimediaDownloadModel = new _converse.MultimediaDownloadModel(downloadable, this.model.attributes.user_id);
+                _converse.multimediaDownloadModel.downloadAttachements();
```

Comment:
This method can go into src/converse-attachement-download.js and then you can use `Object.assign` to assign it to `_converse.ChatBoxView.prototype`